packages/apollo-angular/schematics/install/schema.json

```diff
@@ -1,9 +1,16 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "SchematicsApolloAngularInstall",
-  "title": "Apollo Angular Install Options Schema",
+  "title": "Apollo Angular Install Options SchemaASDSAD",
```

Comment:
Was this intentional?

Reply:
Thanks for the review, both things are solved in #2122